app/views/reservations/_cart_sidebar.html.erb

```diff
@@ -6,20 +6,23 @@
     <h2>Cart</h2><br />
   <% end %>
   <%# allow user to set start/end dates %>
-  <%= render :partial => 'reservations/cart_dates' %>
+  <%= render partial: 'reservations/cart_dates' %>
   <h3>Items</h3>
   <%# list items in cart %>
-  <%= render :partial => 'reservations/list_items_in_cart' %>
-  <hr />
-  <div class="row">
-    <div class="span3" id="cart_buttons">
-    <%= link_to "Empty cart".titleize, url_for(controller: params[:controller], action: 'empty_cart'), :method => :delete, :class => "btn", :id => 'empty_cart_btn', remote: true %>
+  <%= render partial: 'reservations/list_items_in_cart' %>
+  <div id="cart_buttons">
     <%# print Make reservation button %>
     <% if can? :create, Reservation %>
-      <%= link_to sanitize("#{content_tag :i, nil, class: "fa fa-shopping-cart"} #{"Make reservation".titleize}"), new_reservation_path, :class => "btn btn-success" %>
+      <%= link_to new_reservation_path, class: 'btn btn-success' do %>
+        <%= content_tag :i, nil, class: 'fa fa-shopping-cart' %>
+        <%= 'Reserve'.titleize %>
+      <% end %>
     <% else %>
-      <%= link_to sanitize("#{content_tag :i, nil, class: "fa fa-sign-in"} #{"Sign in".titleize}"), new_user_session_path, :class => "btn btn-success" %>
+      <%= link_to new_user_session_path, class: 'btn btn-success' do %>
+        <%= content_tag :i, nil, class: 'fa fa-sign-in' %>
+        <%= 'Sign in'.titleize %>
+      <% end %>
     <% end %>
-    </div>
+    <%= link_to 'Empty cart'.titleize, url_for(controller: params[:controller], action: 'empty_cart'), method: :delete, class: 'btn btn-default', id: 'empty_cart_btn', remote: true %>
```

Comment:
Should we float this right? Nice job switching the order of the buttons, this makes way more sense.